app/src/userpages/components/TransactionPage/List/index.jsx

```diff
@@ -26,188 +19,141 @@ import Meatball from '$shared/components/Meatball'
 import LoadMore from '$mp/components/LoadMore'
 import DocsShortcuts from '$userpages/components/DocsShortcuts'
 import ListContainer from '$shared/components/Container/List'
-import { selectAccountId } from '$mp/modules/web3/selectors'
-import { areAddressesEqual } from '$mp/utils/smartContract'
 import { ago } from '$shared/utils/time'
+import useAccountAddress from '$shared/hooks/useAccountAddress'
+import useCopy from '$shared/hooks/useCopy'
+import useEthereumIdentities from '$shared/modules/integrationKey/hooks/useEthereumIdentities'
+import routes from '$routes'
 
 import styles from './list.pcss'
 
-type StateProps = {
-    fetching: boolean,
-    web3Accounts: ?IntegrationKeyList,
-    transactions: ?TransactionEntityList,
-    products: ProductEntities,
-    hasMoreResults: boolean,
-    accountId: ?Address,
-}
+const TransactionList = () => {
+    const dispatch = useDispatch()
 
-type DispatchProps = {
-    clearTransactionList: () => void,
-    noTransactionResults: () => void,
-    getWeb3Accounts: () => void,
-    getTransactionEvents: () => void,
-    showEvents: () => void,
-    copyToClipboard: (text: string) => void,
-    openInEtherscan: (url: string) => void,
-}
+    const accountId = useAccountAddress()
```

Comment:
I refactored this component to use `useAccountAddress` hook instead of the redux state value.